src/app/pages/request/request.component.ts

```diff
@@ -41,7 +43,7 @@ export class RequestComponent implements OnInit {
   ngOnInit() {
     this.requestForm = this.formBuilder.group({
       priority: ['', Validators.required],
-      title: ['', [Validators.required, Validators.minLength(20)]],
+      title: ['', [Validators.required, Validators.maxLength(40)]],
```

Comment:
this will be for limiting the number of characters. It won't help us on the listing.